src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/RequireThisCheck.java

```diff
@@ -437,6 +437,7 @@ private static boolean isAnonymousClassDef(DetailAST ast) {
      * or null otherwise.
      * @param ast IDENT ast to check.
      * @return the class frame where violation is found or null otherwise.
+     * @noinspection IfStatementWithIdenticalBranches
```

Comment:
@rnveach ! I wasn't sure hence I asked ! I am unable to view the wrecker details actually ! It keeps on giving me a "401 unauthorised". The wrecker.yml roman pointed me out to had a NoExceptionTest and apart from adding a `@noexception` all I did was refactor a few things and change a few comments, so I thought  `@noexception` might be responsible.